prometheus/promhttp/http.go

```diff
@@ -11,21 +11,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Copyright (c) 2013, The Prometheus Authors
-// All rights reserved.
-//
-// Use of this source code is governed by a BSD-style license that can be found
-// in the LICENSE file.
-
-// Package promhttp contains functions to create http.Handler instances to
-// expose Prometheus metrics via HTTP. In later versions of this package, it
-// will also contain tooling to instrument instances of http.Handler and
-// http.RoundTripper.
-//
+// Package promhttp provides new functions for instrumenting go code.
```

Comment:
"new" will seen be "not new anymore". Also, the lead-in for the handler part has slipped further down, where it doesn't make sense.

Suggestion:

"Package promhttp provides functions te create http.Handler instances to expose Prometheus metrics via HTTP. It also provides middleware to instrument HTTP servers and clients."

After this, leave an empty line for a new paragraph.